kratos.gid/apps/Fluid/xml/AnalysisType.spd

```diff
@@ -2,6 +2,6 @@
 <container n="AnalysisType" pn="Analysis Type" help="Analysis information" icon="analysis" state="normal" open_window="0">
     <value n="FreeSurface" pn="Free Surface" un="FLFreeSurface" v="No" values="Yes,No" help="Analysis type" state="hidden"/>
     <value n="SolStrat" pn="Solution Strategy" un="FLSolStrat" v="" values="" dict="[GetSolutionStrategies App Fluid]" actualize_tree="1"/>
-    <value n="Scheme" pn="Scheme" un="FLScheme" v="" values="" dict="[GetSchemes]" actualize_tree="1" state="disabled"/>
+    <value n="Scheme" pn="Scheme" un="FLScheme" v="" values="" dict="[GetSchemes]" actualize_tree="1" state="[getStateFromXPathValue {string(../value[@n='SolStrat']/@v)} Monolithic]"/>

```

Comment:
```suggestion
    <value n="Scheme" pn="Scheme" un="FLScheme" v="" values="" dict="[GetSchemes]" actualize_tree="1" state="[checkStateByUniqueName FLSolStrat Monolithic]"/>

```

Reply:
This way it's independent of the relative path

Reply:
OK. You are the one that knows how this really works :smile: